src/Avalonia.Input/Cursors.cs

```diff
@@ -28,7 +28,7 @@ public enum StandardCursorType
         AppStarting,
         Help,
         TopSide,
-        BottomSize,
+        BottomSide,
```

Comment:
Add
```cs
[Obsolete("Use BottomSide")]
ButtomSize = BottomSide,

```

Reply:
typo should be BottomSize

Reply:
@kekekeks This enum without number that we should not add value optionally.

Reply:
@ahopper @lindexi we shouldn't just break people's code by randomly renaming stuff. The common practice is to make the old naming obsolete and remove it after several versions.

Reply:
very true but you have spelled BottonSize as ButtomSize

Reply:
Ah, ok